app.go

```diff
@@ -333,7 +338,9 @@ func (a *App) RunAsSubcommand(ctx *Context) (err error) {
 	// append help to commands
 	if len(a.Commands) > 0 {
 		if a.Command(helpCommand.Name) == nil && !a.HideHelp {
-			a.appendCommand(helpCommand)
+			if !a.HideHelpCommand {
```

Comment:
Looks like the reason the PR checks are failing is because this condition isn't being checked anywhere in the tests. See: https://codecov.io/gh/urfave/cli/pull/1083

Reply:
Added tests for `RunAsSubcommand`, but codecov is still failing

Reply:
now green